pxd.c

```diff
@@ -1132,7 +1132,7 @@ int pxd_init(void)
 out_misc:
 	misc_deregister(&pxd_miscdev);
 out_fuse:
-	for (j = 0; j < i; ++j) {
+	for (j = 0; j <= i; ++j) {
```

Comment:
Maybe instead the init function should be refactored so it wouldn't be allowed to fail?